tool_commands/constants.py

```diff
@@ -0,0 +1,29 @@
+from pathlib import Path
+
+
+COMMANDS_DIR = Path(__file__).parent
+
+modulename_packagename_mapping = {
+    "raster_checker.py": "raster_checker",
+    "schematisation_checker.py": "schematisation_checker",
+    "import_sufhyd.py": "import_sufhyd",
+    "control_structures.py": "control_structures",
```

Comment:
De namen zijn hier allemaal lekker consistent. Kan je dit dan niet automatisch doen? Dan dwing je gelijk af dat de naam hetzelfde is.
Of je doet `raster_checker/command.py` i.p.v. `raster_checker/raster_checker.py`?

Het lijkt ook meer een filename-to-packagename-mapping?